spec/features/reports/interval_cdrs/index_interval_cdr_items_spec.rb

```diff
@@ -7,13 +7,23 @@
 
   include_context :login_as_admin
 
-  let!(:report) { create(:interval_cdr) }
-  let!(:interval_data) { create(:interval_data, report: report) }
+  let!(:report) do
+    FactoryBot.create(:interval_cdr, group_by: %w[customer_id vendor_id rateplan_id disconnect_initiator_id])
+  end
+  let!(:interval_data) do
+    FactoryBot.create(:interval_data, rateplan: rateplan, report: report, customer: customer, vendor: vendor)
+  end
+  let(:customer) { FactoryBot.create(:customer) }
+  let(:vendor) { FactoryBot.create(:vendor) }
+  let(:rateplan) { FactoryBot.create(:rateplan) }
 
   it 'should have table with interval data and destination rate policy' do
     subject
     expect(page).to have_table
     within_table_row(id: interval_data.id) do
+      expect(page).to have_table_cell(text: customer.display_name, column: 'Customer')
+      expect(page).to have_table_cell(text: vendor.display_name, column: 'Vendor')
+      expect(page).to have_table_cell(text: rateplan.display_name, column: 'Rateplan')
```

Comment:
add test with all possible group_by columns that are foreign keys to other tables. add same tests for `custom_cdrs` as well